app/src/androidTest/java/org/mozilla/fenix/ui/robots/HomeScreenRobot.kt

```diff
@@ -233,54 +222,6 @@ class HomeScreenRobot {
             )
     }
 
-    fun verifySelectTabsView(vararg tabTitles: String) {
```

Comment:
Had to make some changes/cleanup to the homescreen collections and tabs old code.